internal/monitors/aspdotnet/metrics.go

```diff
@@ -1,20 +1,20 @@
 package aspdotnet
 
-// COUNTER(asp_net.application_restarts): Count of ASP.NET application restarts.
+// GAUGE(asp_net.application_restarts): Count of ASP.NET application restarts.
 
 // GAUGE(asp_net.applications_running): Number of running ASP.NET applications.
 
 // GAUGE(asp_net.requests_current): Current number of ASP.NET requests.
 
 // GAUGE(asp_net.requests_queue): Number of queued ASP.NET requests.
 
-// COUNTER(asp_net.requests_rejected): Count of rejected ASP.NET requests.
+// GAUGE(asp_net.requests_rejected): Count of rejected ASP.NET requests.
```

Comment:
What is the rationale for making them gauges?  Is there the potential to "miss" some of the rejected requests?

Reply:
It's what the original perf counter reporter does.  Everything is a gauge.  And the docs on these counters aren't the greatest.